codemagic.yaml

```diff
@@ -0,0 +1,99 @@
+definitions:
+  configure_environment: &configure_environment
+    instance_type: mac_mini_m1
+    max_build_duration: 60
+    cache:
+      cache_paths:
+        - $HOME/.gradle/caches
+  scripts:
+    - &detekt_on_template_compose
+      name: Run detekt on template-compose
+      working_directory: ./template-compose
+      script: ./gradlew detekt
+    - &detekt_on_template_xml
+      name: Run detekt on template-xml
+      working_directory: ./template-xml
+      script: ./gradlew detekt
+    - &unit_test_on_template_compose
+      name: Run unit tests on template-compose
+      working_directory: ./template-compose
+      script: ./gradlew koverMergedReport
+    - &unit_test_on_template_xml
+      name: Run unit tests on template-xml
+      working_directory: ./template-xml
+      script: ./gradlew koverMergedReport
```

Comment:
I'm wondering do we have an option to initialize Codemagic, Bitrise, or Github Action workflows when generating the new project from the script. cc @nimblehq/android-chapter

Reply:
@lydiasama Any thoughts? or raising to the TL 😛

Reply:
Yep, let's try to automate everything we can instead of cloning or copying-paste 😄